ambari-server/src/test/java/org/apache/ambari/server/security/authentication/jwt/AmbariJwtAuthenticationFilterTest.java

```diff
@@ -125,14 +125,13 @@ private SignedJWT getSignedToken(Date expirationTime, String audience) throws JO
 
     Calendar calendar = Calendar.getInstance();
     calendar.setTimeInMillis(System.currentTimeMillis());
-    JWTClaimsSet claimsSet = new JWTClaimsSet();
-    claimsSet.setSubject("test-user");
-    claimsSet.setIssuer("unit-test");
-    claimsSet.setIssueTime(calendar.getTime());
-
-    claimsSet.setExpirationTime(expirationTime);
-
-    claimsSet.setAudience(audience);
+    JWTClaimsSet claimsSet = new JWTClaimsSet.Builder()
```

Comment:
Was this change needed?

Reply:
yes, one of the dependency change caused a compilation error